mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule AppStore.MixProject do
   use Mix.Project
 
-  @version "0.1.0"
```

Comment:
Hi @mrnovalles ,

Thanks for the PR, can you please roll back the change to the version number? I'll release a 0.2.0 soon with other improvements.

Reply:
I appreciated adding the changes to CHANGELOG.md as well. 🍻

Reply:
sure, I'll do that, thanks :)